.github/workflows/nodejs.yml

```diff
@@ -12,6 +12,18 @@ on:
       - next
       - v4
 
+  workflow_call:
+    inputs:
+      update-snapshots:
+        description: "Update snapshots?"
+        type: boolean
+
+  workflow_dispatch:
+    inputs:
+      update-snapshots:
+        description: "Update snapshots?"
+        type: boolean
```

Comment:
Let's not add this. Tests should ideally fail if snapshots are not in sync with the master branch, this would let user bypass failing CI.